lxd/instance/instance_utils.go

```diff
@@ -682,7 +682,7 @@ func ValidName(instanceName string, isSnapshot bool) error {
 		}
 	} else {
 		if strings.Contains(instanceName, shared.SnapshotDelimiter) {
-			return fmt.Errorf("The character %q is reserved for snapshots", shared.SnapshotDelimiter)
+			return fmt.Errorf("Invalid instance name: Cannot contain slashes")
```

Comment:
To be in line with the other messages, would it make sense to add the name of the instance to the error message too?

Reply:
yeah i think it would

Reply:
Fixed in https://github.com/canonical/lxd/pull/13261/commits/1009cf770710c4ba3c9f437d1daaf4388c9e80b3